src/test/scala/org/broadinstitute/dsde/agora/server/webservice/ApiServiceSpec.scala

```diff
@@ -29,6 +29,13 @@ class ApiServiceSpec extends FlatSpec with Matchers with Directives with Scalate
     }
   }
 
+  "Agora" should "return methods matching the query" in {
+    agoraDao.insert(testEntity2)
+    Get(ApiUtil.Methods.withLeadingSlash + "?namespace=broad") ~> methodsService.queryByEntityRoute ~> check {
```

Comment:
Do we want to test search with more than one parameter?
Probably namespace and owner (maybe want to remove the spaces from testMethod owner